### ISSUE TITLE
Pytorch BatchNorm1d change to onnx,have Squeeze and Unsqueeze op.How to simplify them?

### ISSUE THREAD

**Issue by yangchengtest** [daquexian/onnx-simplifier#12]
Code:
 nn.BatchNorm1d(filter_list[4] * 3 * 3)
 
Onnx net:
Unsqueeze->BN->Squeeze.

Ncnn not support Unsqueeze, How can I simplify them?
Thanks~

**Comment by daquexian** [comment]
@yangchengtest ONNX BatchNorm op has support N-d array (instead only 4d array) since opset 6 (https://github.com/onnx/onnx/blob/master/docs/Changelog.md#BatchNormalization-6). However, PyTorch has not updated its code. I'll propose a PR to PyTorch repo.

**Comment by daquexian** [comment]
https://github.com/pytorch/pytorch/pull/22857

**Comment by daquexian** [comment]
It looks like the PR above will not be merged soon.

Please edit `torch/onnx/symbolic_opset9.py` in your pytorch package directory (run `python -c "import torch; print(torch.__file__)"` to get the directory path) following https://github.com/pytorch/pytorch/pull/22857/files to solve this problem manually

**Comment by yangchengtest** [comment]
Thanks for your codes.~
We have compiled pytorch with your commit,and slove the problem.~

> 
> 
> It looks like the PR above will not be merged soon.
> 
> Please edit `torch/onnx/symbolic_opset9.py` in your pytorch package directory (run `python -c "import torch; print(torch.__file__)"` to get the directory path) following https://github.com/pytorch/pytorch/pull/22857/files#diff-fbdd288f9f129017c8de36ead0f6e5b3 to solve this problem manually

**Comment by daquexian** [comment]
> Thanks for your codes.~
> We have compiled pytorch with your commit,and slove the problem.~
> 
> > It looks like the PR above will not be merged soon.
> > Please edit `torch/onnx/symbolic_opset9.py` in your pytorch package directory (run `python -c "import torch; print(torch.__file__)"` to get the directory path) following https://github.com/pytorch/pytorch/pull/22857/files#diff-fbdd288f9f129017c8de36ead0f6e5b3 to solve this problem manually

@yangchengtest Glad to see it. However, you don't need to re-compile pytorch (because what you edited is only python files) :D

**Comment by nihui** [comment]
插眼关注下
